app/assets/stylesheets/application.scss

```diff
@@ -59,6 +59,7 @@ $govuk-use-legacy-palette: false;
 @import "views/report-child-abuse";
 @import "views/local-authority";
 @import "views/simple-smart-answer";
+@import "views/country-notice";
```

Comment:
The organisation of this file doesn't seem quite right to me, since the country notice is a sub view of transaction rather than a view in its own right. Though looking at the rest of the file the organisation of CSS in this app looks a bit of a car crash 🙈 .

I imagine the simplest thing would be to include this styling on a views/transaction stylesheet that maps to any additional transaction styling.